sdks/java/core/pom.xml

```diff
@@ -121,6 +121,13 @@
         </executions>
       </plugin>
 
+      <!-- Ensure that the Maven jar plugin runs before the Maven
+        shade plugin by listing the plugin higher within the file. -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I mean that the fact that we configured the shade plugin in the `plugins` area is a bug. It cause the order to be changed.